swift/Sources/Addresses/CoinType+Address.swift

```diff
@@ -60,8 +63,8 @@ public extension CoinType {
         case .iocoin:
             return IocoinAddress(string: string)
         case .groestlcoin:
```

Comment:
we can merge this into `.bitcoin` case

Reply:
`.groestlcoin` and `.bitcoin` are a bit different

```
            if let segwitAddress = SegwitAddress(string: string) {
                return segwitAddress
            } else {
                return GroestlcoinAddress(string: string)
            }
```

Reply:
oh ok